.github/actions/clean-vpc/gcp.go

```diff
@@ -0,0 +1,50 @@
+package main
+
+import (
+	"context"
+	"fmt"
+	"log"
+	"os"
+	"strings"
+
+	"github.com/mongodb/mongodb-atlas-kubernetes/test/e2e/config"
+	"github.com/mongodb/mongodb-atlas-kubernetes/test/e2e/utils"
+
+	"google.golang.org/api/compute/v1"
+)
+
+func deleteGCPVPCBySubstr(gcpProjectID, nameSubstr string) (bool, error) {
+	ok := true
+	computeService, err := compute.NewService(context.Background())
+	if err != nil {
+		return false, fmt.Errorf("failed to create compute service: %s", err)
+	}
+	networkService := compute.NewNetworksService(computeService)
+	networks, err := networkService.List(gcpProjectID).Do()
+	if err != nil {
+		return false, fmt.Errorf("failed to list networks: %s", err)
+	}
+	for _, network := range networks.Items {
+		if strings.Contains(network.Name, nameSubstr) {
```

Comment:
despite it being a minor problem once we run the test in a dedicated "safe" environment, wouldn't it be better if we update our test to have a fixed prefix instead? and rely on it to identify rather than substring?
As there's no enforce for pattern in the test, have the prefix would also ensure we always identify resources created by the tests

Reply:
Done